api-reference/10 UI Components/dxHtmlEditor/3 Methods/blur().md

```diff
@@ -3,7 +3,9 @@ id: dxHtmlEditor.blur()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+<!-- %shortDescription% -->
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+<!-- %fullDescription% -->
+
+<!-- import * from 'api-reference\10 UI Components\dxTextEditor\3 Methods\blur().md' -->
```

Comment:
It is necessary to discuss the interaction of the 'blur' method with html editor toolbar